<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_nomDuFichier">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_nomDuFichier">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom du fichier : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_build">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Build</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_aps">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>aps</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_aps_vector">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_fs_vector">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_distance_titre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Distance</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_sommet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sommet de d√©part : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_distance">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_parcourir">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parcourir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_dist">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>dist</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dist_vector">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_distance_titre_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Rang</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_rang">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ddi : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rang_vector">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_tarjan_titre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Tarjan</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_prem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>prem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_pilch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pilch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_cfc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>cfc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_prem_vector">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pilch_vector">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_num">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>num</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cfc_vector">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_vector">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
